pagopa/pn-frontend-e2e
# Pn 9334 e 2 e destinatario altri recapiti visualizzazione già associati PF

## Short description

added checks on table "Già associati", modified step to make it viable for both PG and PF

## List of changes proposed in this pull request

- implemented checks on the table's fields PF
- modified 9162's last step to make it viable for both PG and PF
- some method's name refactor

## How to test

Simply run `mvn clean test -Dcucumber.filter.tags=@TA_visualizzazioneSezioneAltriRecapitiPF` and verify if the test works fine.
